android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,8 +1,4 @@
-• OpenStreetMap data as of May 28
-• search for US zip codes and UK (GB) postcodes
-• street name announcements in verbal TTS turn instructions (please enable it in settings), thanks to Will Bradley
-• export bookmarks and tracks in GPX file format
-• save bookmarks and tracks to a local device storage
-• add, edit, and display web links to menus of restaurants, bars, and other food services
+This is a minor interim release just to keep your maps updated!
+• OpenStreetMap data as of June 13
```

Comment:
@pastk can you please restore previous release notes and only change the OSM date and add a line about a fixed bug for iOS? It is critically important to tell all new and existing users about the major release features (iCloud, street announcement, etc.). By deleting previous release notes users won't discover it in any way. That was the reason I told you in the discussion to leave the previous release notes.

Reply:
@biodranik I don't really like this approach, I think its confusing. There is no obvious hint how the minor release is different from a full one, except the date change (which is not easy to notice).

Its like "let's keep major changes repeating in relnotes for several months to make emphasis on them" :)

I've added a link to the prev full release notes (`…see changes from the last full-blown release at omaps.org/news`) so its easy to discover recent bigger changes too.

Reply:
It's not about liking or not liking. It is about letting users know about project progress/features.

Did you investigate how it is done in other apps in stores? Many of them are keeping the major release notes for several hotfix updates. That makes perfect sense for everyone.

Reply:
I don't like when apps "keep the major notes for several hotfix updates". For me it defeats the purpose of relnotes, it makes it hard to see what has really changed in this particular update and hence e.g. its hard to decide whether I want to update or not.
IMO if users will see same notes over and over again then their value will be diminished and users will pay less attention to them next time. Anybody who likes when they're tricked to read the same stuff over again?

Reply:
Looks like you've never seen how it is done (and should be done). Example:

* Hotfixed the bug with app hanging
* New OSM map data as of today

Other important changes in the June release:
* Enabled iCloud
* Added GPX export
* Enable street announcements in settings and enjoy!
* ...

Reply:
Otherwise, following your approach, the hotfix on the next day after a release should delete all longer release texts in stores. It doesn't make any sense.